src/main/external-resources/UMS.conf

```diff
@@ -1410,6 +1410,7 @@ logging_filter_console =
 # get back things filtered out by Log Level above, but you can further filter
 # out more before it's written to the log window. Valid values are:
```

Comment:
@Sami32 it seems there's some redundancy on this line too, can you clean that up while you're at it?